.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,14 @@
+name: golangci-lint
+on:
+  pull_request:
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

I'm not familar with this action, but it seems `v3` is released here as well: https://github.com/marketplace/actions/run-golangci-lint

```suggestion
        uses: golangci/golangci-lint-action@v3
```